odoo/modules/loading.py

```diff
@@ -431,3 +431,26 @@ def load_modules(db, force_demo=False, status=None, update_module=False):
             _logger.log(25, "All post-tested in %.2fs, %s queries", time.time() - t0, odoo.sql_db.sql_counter - t0_sql)
     finally:
         cr.close()
+
+
+def cleanup_modules(db):
```

Comment:
function name is not really meaningful. Something like `reset_modules_state` would be better